js/siteorigin-panels/view/widget.js

```diff
@@ -20,8 +20,13 @@ module.exports = Backbone.View.extend( {
 	 * Initialize the widget
 	 */
 	initialize: function () {
-		this.model.on( 'user_edit', this.editHandler, this );				 // When a user wants to edit the widget model
-		this.model.on( 'user_duplicate', this.duplicateHandler, this );	   // When a user wants to duplicate the widget model
+		// When a user wants to edit the widget model
+		this.model.on( 'user_edit', this.editHandler, this );
```

Comment:
It's better to use `this.listenTo( this.model, 'user_edit', this.editHandler );` as it allows all listeners to be removed more easily and removes the need for providing `this` as context every time.

Reply:
Awesome, will change that.